CHANGELOG-Enterspeed.Source.UmbracoCms.md

```diff
@@ -4,13 +4,18 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.3.2 - 2025-05-07]
+### Fixed
+- Deleting more than 2,100 failed jobs caused an error.
+- The RTE field converter threw a null exception when run as a background job.
+
 ## [5.3.1 - 2025-02-17]
 ### Fixed
-- Fixed culture handling when rename of node should update urls on all descendants in Enterspeed sources.
+- Fixed culture handling when rename of node should update urls on all descendants in Enterspeed sources. (contribution by [Joao Diogo Vicente Henrques](https://github.com/JoaoDiogoVicenteHenrques))
 
 ## [5.3.0 - 2025-01-27]
 ### Added
-- Added JSON schema file to provide IntelliSense in appsettings.json for Enterspeed settings
+- Added JSON schema file to provide IntelliSense in appsettings.json for Enterspeed settingsd q
```

Comment:
Some typos?

Reply:
I have no idea how that decided to happen.